src/main/java/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package service.impl;
```

Comment:
The package declaration should not be followed by an empty line. Please remove the empty line after the package statement.